dev/doc/xml-protocol.md

```diff
@@ -151,16 +151,15 @@ state that should become the next tip.
 * Failure:
   - Syntax error. Error offsets are byte offsets (not character offsets) with respect to the start of the sentence, starting at 0.
```

Comment:
Should probably mention that _all_ of the field values received may need to be adjusted if the sentence was have moved since it was sent to Coq (e.g. due to async mode).  Maybe mention byte offsets vs UTF-8 character offsets/conversion.  Extra credit if you explain how to adjust values.